templates/app/commercial-vendor-program.phtml

```diff
@@ -0,0 +1,77 @@
+<?php
+const CVP_CONTACT_EMAIL = 'commercial-vendor-program@getlaminas.org';
```

Comment:
This part we need to sort; we don't actually have an email provider at this point. I have our domain registrar forwarding the security email to me, but I only get one free forward. We should likely look into something like groups.io or similar, depending on our organization funds.